packages/core/src/plugins/algolia.js

```diff
@@ -24,11 +24,11 @@ function addNoIndexClasses(content) {
     'dropdown',
     'b-modal',
     'panel:not([expanded])',
-    'popover div[slot=content]',
+    'span[data-mb-component-type="popover"] div[data-mb-slot-name="content"]',
```

Comment:
let's generalise this so tooltips get the fix too, also non `<div>` elements

Reply:
Thanks for the review! Made the changes as per our discussion :-)